docs/src/man/hosting.md

```diff
@@ -267,6 +267,16 @@ to the configuration file, as showed in the [previous section](@ref GitHub-Actio
     of the deployment is the same as the current repository. In order to push
     elsewhere you should instead use a SSH deploy key.
 
+#### Permissions
+
+The following [GitHub Actions job or workflow permissions](https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs) are required to successfully use [`deploydocs`](#the-deploydocs-function):
+
+```yaml
+permissions:
+  contents: write
```

Comment:
Is this required also if you have a deploy key?

Reply:
This is in the `Authentication: GITHUB_TOKEN` section though, so I think it's appropriate?

Reply:
From the GitHub actions documentation I linked:

> You can use `permissions` to modify the default permissions granted to the `GITHUB_TOKEN`, adding or removing access as required, so that you only allow the minimum required access.

SSH deploy keys can only be used for Git repo access. Since `push_preview=true` [requires `pull-requests: true`](https://github.com/JuliaDocs/Documenter.jl/blob/346c622bf7eb7bc70ab7de6481650285de599921/src/deployconfig.jl#L526) that would also be true there

Reply:
I've moved this to be a section after "Authentication: SSH deploy keys" as it's relevant to both `GITHUB_TOKEN` and SSH deploy keys